source/_integrations/google_assistant_sdk.markdown

```diff
@@ -36,6 +36,8 @@ This integration allows:
   - What does the elephant say?
 - Having a conversation with Google Assistant using the [conversation](/integrations/conversation/) integration via text or voice.
 
+Note: If you want to get responses from commands to Google Assistant in automations install the [Google Assistant SDK Custom](https://github.com/tronikos/google_assistant_sdk_custom) integration.
```

Comment:
Sorry, we don't refer to custom integrations, as the Home Assistant project does not want to recommend using them or endorse. If you want to  have a more central location to advertise/put out your custom integration, I suggest to get it listed in HACS.